<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playbackWindow</class>
 <widget class="QWidget" name="playbackWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="PlaybackSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>53</width>
     <height>481</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdPlayForward">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>▶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdPlayReverse">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>◀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdSave">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdSaveSettings">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdMarkIn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Mark
Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdMarkOut">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Mark
End</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>70</y>
     <width>141</width>
     <height>96</height>
    </rect>
   </property>
   <property name="title">
    <string>Play Rate</string>
   </property>
   <widget class="QLabel" name="lblFrameRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdRateDn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdRateUp">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdLoop">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>431</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCurrentFrame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFrame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaybackSlider</class>
   <extends>QSlider</extends>
   <header>playbackslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
